extensions/WEBGL_lose_context/extension.xml

```diff
@@ -29,7 +29,7 @@
     </p>
   </overview>
   <idl xml:space="preserve">
-[LegacyNoInterfaceObject]
+[Exposed=Window, LegacyNoInterfaceObject]
```

Comment:
@kenrussell This should be `Exposed=(Window,Worker)`, right?

Reply:
Yes. We'll have to go through these automated pull requests one by one and fix them up manually - if you don't mind taking this one would certainly appreciate it.